src/pages/home/report/ReportActionCompose.js

```diff
@@ -463,8 +463,9 @@ class ReportActionCompose extends React.Component {
             return null;
         }
 
-        const reportParticipants = lodashGet(this.props.report, 'participants', []);
-        const reportRecipient = this.props.personalDetails[reportParticipants[0]];
+        const reportParticipants = _.without(lodashGet(this.props.report, 'participants', []), this.props.myPersonalDetails.login);
+        const participantsWithoutExpensifyEmails = ReportUtils.removeExpensifyEmails(reportParticipants);
+        const reportRecipient = this.props.personalDetails[participantsWithoutExpensifyEmails[0]];
```

Comment:
Same here with handling possibly null case

Reply:
As we are already returning false when participant isn't there so shouldShowReportRecipientLocalTime will be false and won't be rendering the `ParticipantLocalTime`
https://github.com/Expensify/App/blob/5b0cd481c87e89772f7d21fdbad6bf815b495b15/src/pages/home/report/ReportActionCompose.js#L485-L486

Reply:
Same as above, this might be fine since participantsWithoutExpensifyEmails will be an empty array